packages/slate-react/test/index.spec.tsx

```diff
@@ -95,4 +95,49 @@ describe('slate-react', () => {
       })
     })
   })
+
+  test('calls onSelectorChange when editor select change', async () => {
+    const editor = withReact(createEditor())
+    const initialValue = [
+      { type: 'block', children: [{ text: 'te' }] },
+      { type: 'block', children: [{ text: 'st' }] },
+    ]
+    const onSelectorChange = jest.fn()
+    act(() => {
+      create(
+        <Slate
+          editor={editor}
+          initialValue={initialValue}
+          onSelectorChange={onSelectorChange}
+        >
+          <Editable />
+        </Slate>,
+        { createNodeMock }
+      )
+    })
+    await act(async () =>
+      Transforms.select(editor, { path: [0, 0], offset: 2 })
+    )
+    expect(onSelectorChange).toHaveBeenCalled()
+  })
+
+  test('calls onValueChange when editor children change', async () => {
+    const editor = withReact(createEditor())
+    const initialValue = [{ type: 'block', children: [{ text: 'test' }] }]
+    const onValueChange = jest.fn()
+    act(() => {
+      create(
+        <Slate
+          editor={editor}
+          initialValue={initialValue}
+          onValueChange={onValueChange}
+        >
+          <Editable />
+        </Slate>,
+        { createNodeMock }
+      )
+    })
+    await act(async () => Transforms.insertText(editor, 'Hello word!'))
+    expect(onValueChange).toHaveBeenCalled()
+  })
```

Comment:
Could you add 
```ts
expect(onChange).toHaveBeenCalled()
expect(onSelectionChange).toHaveBeenCalled()
```

And another test using `Transforms.setNodes` that should not change selection?

Reply:
I added a unit test case for setting bold using Transforms.setNodes.